db/migrate/20151435234635_create_container_volumes.rb

```diff
@@ -0,0 +1,49 @@
+class CreateContainerVolumes < ActiveRecord::Migration
+  def up
+    create_table :container_volumes do |t|
```

Comment:
For now all columns are included but as far as I understand some of them (e.g. ```:gce_fs_type```, ```:aws_fs_type```, ```:iscsi_fs_type```,```:rbd_fs_type```,  ```:cinder_fs_type```) may be dropped.